decidim-assemblies/app/controllers/decidim/assemblies/admin/assembly_landing_page_content_blocks_controller.rb

```diff
@@ -29,6 +31,12 @@ def edit_resource_landing_page_path
         def resource_landing_page_content_block_path
           assembly_landing_page_content_block_path(scoped_resource, params[:id])
         end
+
+        alias current_participatory_space current_assembly
+
+        def parent_assembly
+          current_assembly.parent
+        end
```

Comment:
The entire controller is referring to `scoped_resource`. I would prefer to have only an object that we use.
```suggestion

        def parent_assembly
          scoped_resource.parent
        end
```